chapters/operatorsandexpressions.tex

```diff
@@ -359,8 +359,8 @@ \subsection{Member Access Operator}\label{member-access-operator}
 
 \subsection{Built-in Variable time}\label{built-in-variable-time}\indexinline{time}
 
-All declared variables are functions of the independent variable \lstinline!time!.
-The variable \lstinline!time! is a built-in variable available in all models and blocks, which is treated as an input variable.
+All declared variables are functions of \lstinline!time!.
+The name \lstinline!time! is a keyword see \cref{modelica-keywords}, but it corresponds to a built-in variable available in all models and blocks, which is treated as an input variable.
```

Comment:
Missing parenthesis or comma, and reformulate without _but_ to sound less like an oddity:
```suggestion
The name \lstinline!time! is a keyword (see \cref{modelica-keywords}), and corresponds to a built-in variable available in all models and blocks, which is treated as an input variable.
```